cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -166,9 +166,16 @@ def test_change_cpu_ram(provisioner, soft_assert, provider, prov_data, vm_name):
 @pytest.mark.rhv3
 # Special parametrization in testgen above
 @pytest.mark.meta(blockers=[1209847, 1380782])
-@pytest.mark.parametrize("disk_format", ["Thin", "Thick", "Preallocated"])
-@pytest.mark.uncollectif(lambda provider, disk_format:
-                         (provider.one_of(RHEVMProvider) and disk_format == "Thick") or
+@pytest.mark.meta(automates=[1633867])
+@pytest.mark.parametrize("disk_format", ["Thin", "Thick", "Preallocated",
+    "Thick - Lazy Zero", "Thick - Eager Zero"])
```

Comment:
TODO: Eager zero will require the max retries to be modified. Will add it once the build with this feature is ready.